draft-ietf-tls-extended-key-update.md

```diff
@@ -448,14 +450,16 @@ has the following steps:
    `ExtendedKeyUpdate(key_update_response)` message. While an extended key update
    is in progress, the responder MUST NOT initiate further key updates. The responder MAY defer
    sending a response if system load or resource constraints prevent immediate processing.
-   In such cases, responder MUST confirm receipt of the the key_update_request with an ACK and
-   send an actual key_update_response once sufficient resources become available.
+   In such cases, responder MUST confirm receipt of the key_update_request with an ACK and
```

Comment:
nit: maybe "In such cases, the responder MUST acknowledge receipt of the key_update_request with an ACK and, once sufficient resources become available, retransmit the key_update_response until it is acknowledged by the initiator.". 

Or does it change the logic?